recod_frappe_devtools/build_docs/setup_docs.py

```diff
@@ -0,0 +1,368 @@
+"""Automatically setup docs for a project
+
+Call from command line:
+
+	bench setup-docs app path
+
+"""
+from __future__ import unicode_literals, print_function
+
+import os, json, frappe, shutil
+
+import jinja2
+from frappe.utils import markdown
+
+
+# def is_singe_doc(basepath, doctype):
+#     if "__" not in doctype:
+#         with open(basepath + "/" + doctype + ".json", 'r') as f:
+#             data_str = f.read()
+#             data = json.loads(data_str)
+#             return data['issingle'] == 1
+#     return True
+
+
+class SetupDocs(object):
+    def __init__(self, app, target_app):
+
+        self.list_sidebar = []
+        self.app = app
+        self.target_app = target_app
+
+        frappe.flags.web_pages_folders = ['docs', ]
+        frappe.flags.web_pages_apps = [self.app, ]
+        self.list_without_app_documentation = ['frappe', 'erpnext']
```

Comment:
Do not hardcode the list of apps to exclude
```suggestion
        self.list_without_app_documentation = []
```